packages/as-types/assembly/serialization/bool.ts

```diff
@@ -12,7 +17,9 @@ export function byteToBool(arr: StaticArray<u8>): bool {
 /**
  * Converts a boolean to StaticArray<u8>.
  *
```

Comment:
@remarks

 * The conversion is performed by creating a `StaticArray<u8>` of length 1, where the first byte represents the `boolean` value. If the value is `true`, the byte is set to 1, otherwise, it is set to 0.